confgenerator/fluentbit/modify.go

```diff
@@ -0,0 +1,96 @@
+package fluentbit
```

Comment:
@quentinmit - Since you are working on a generic`modify_fields` processor, could you help take a look at this file to see if there are any concerns?

Reply:
These look like helpers for generating fluentbit config directly. I'm not particularly worried about what that API looks like - whatever makes the processors simple and readable is good. I doubt we'll use any of this code for the generic processor implementation, though.